service/service_mocks.go

```diff
@@ -0,0 +1,85 @@
+package service
```

Comment:
Only `mock_*.go` and `*_mock.go` are being excluded from Sonar coverage. This will drop the test coverage stats.